### DISCUSSION TITLE
Lesson 10 brownie compile error

### DISCUSSION THREAD

**Discussion by delphee** [smartcontractkit/full-blockchain-solidity-course-py#467] [Q&A] [answered]
CompileError:  solc returned the following errors:
Invalid remapping: "= aave=C:/Users/username/.brownie/packages/= aave"

Why is it trying to remap to a file on my computer?  I have brownie-config.yaml correct to the letter, and the scripts as well.

**Comment by cromewar** [comment, marked as answer]
You may have a identation error on your brownie-config try to import that file directly from github.

**Reply by delphee** [reply]
If I replace "@aave" with "aave/protocol-v2@1.0.1" in ILendingPool.sol, I get the same error.

**Reply by cromewar** [reply]
Could you please share a screenshot of your brownie config and your project folder structure?

**Reply by delphee** [reply]
Sure: 
![image](https://user-images.githubusercontent.com/38186979/144225565-98e183fb-b377-4122-a6f9-7f25d1d5298a.png)

**Reply by cromewar** [reply]
It seems to be a problem with your aave installation, but before anything make sure your imports look like this:
```Solidity

import {ILendingPoolAddressesProvider} from '@aave/contracts/Interfaces/ILendingPoolAddressesProvider.sol';
import {DataTypes} from '@aave/contracts/protocol/libraries/types/DataTypes.sol';
```
Now delete the build folder on your project and try again.

If nothing works you would like to delete aave files from your brownie installation, go to your User Folder on `C:/Users/username/.brownie` inside the packages folder delete the aave folder and try again.

**Reply by delphee** [reply]
Thank you, I will try this later today and report back.

**Reply by delphee** [reply]
I tried deleting the build folder.
After deleting the aave folders, it did work. I got this error, however:

WARNING: Unable to compile aave/protocol-v2@1.0.1 due to a CompilerError - you may still be able to import sources from the package, but will be unable to load the package directly.

Not sure what that is about, but the transaction went through. Once again, thank you sir!

**Reply by cromewar** [reply]
![image](https://user-images.githubusercontent.com/20199180/144336580-209ca4bc-d4ad-478a-a6a5-63b67dcfeb60.png)
 
As you can see on the image above, you have the Interfaces you copied from github, so brownie is saying you that the same `.sol` contracts exists both on github and on your project, so the warning is saying it won't use both, but you don't have to worry about.

As you problem is solved, I'll close this thread now. Feel free to ask any other. 

Best regards, Cromewar.
